adapters/zwave/zwave-classifier.js

```diff
@@ -13,20 +13,21 @@
 // See; http://wiki.micasaverde.com/index.php/ZWave_Command_Classes for a
 // complete list of command classes.
 
-const COMMAND_CLASS_SWITCH_BINARY = 37;
-const COMMAND_CLASS_SWITCH_MULTILEVEL = 38;
-const COMMAND_CLASS_SWITCH_ALL = 39;
+const COMMAND_CLASS_SWITCH_BINARY = 37;     // 0x25
+//const COMMAND_CLASS_SWITCH_MULTILEVEL = 38; // 0x26
+//const COMMAND_CLASS_SWITCH_ALL = 39;        // 0x27
```

Comment:
I commented them out because the linter was complaining that they're not being used. Which is true for now. They will be used soon as we add more support for more types of things.